man/org.freedesktop.systemd1.xml

```diff
@@ -277,6 +277,10 @@ node /org/freedesktop/systemd1 {
       GetDynamicUsers(out a(us) users);
       DumpUnitFileDescriptorStore(in  s name,
                                   out a(suuutuusu) entries);
+      StartAuxiliaryScope(in  s name,
+                          in  ah pidfds,
+                          in  t flags,
```

Comment:
this should also have `in a(sv) properties` here, like StartTransientUnit(), in case people want to set some prop explicitly for the scope unit. for example, i'd recommend them to set the description and similar.

We have the parser for the props anyway, hence it's a trivial additional, and the people who don#t want to use it can just pass an empty array, np

Reply:
Added.